wasm/src/language_service.rs

```diff
@@ -486,11 +498,13 @@ serializable_type! {
     NotebookMetadata,
     {
         pub targetProfile: Option<String>,
-        pub languageFeatures: Option<Vec<String>>
+        pub languageFeatures: Option<Vec<String>>,
+        pub manifest: Option<Manifest>,
     },
     r#"export interface INotebookMetadata {
         targetProfile?: "unrestricted" | "base";
         languageFeatures?: "v2-preview-syntax"[];
+        manifest?: string;
     }"#,
     INotebookMetadata
```

Comment:
The types of `manifest` don't match between the Rust and the TypeScript declaration here, which makes me think there might be some type error happening when this actually gets used.

Reply:
See https://github.com/microsoft/qsharp/blob/e3fa6744610bc253365f0c8372e27721d0e0f8ac/vscode/src/notebook.ts#L166
 
This function is throwing at the wasm-bindgen boundary. I'm guessing it receives a string when it expects a `Manifest` object.

Tested with a notebook and qsharp.json like this:

![image](https://github.com/microsoft/qsharp/assets/16928427/14ec4b2b-f30f-4aaf-babf-9cc280d3edbf)

Reply:
Thanks! I fixed the type mismatch. I will reach out for help with the integration test.